resources/views/layouts/base.blade.php

```diff
@@ -44,6 +44,8 @@
 
 @stack('modals')
 
+@vite(['resources/js/app.js'])
```

Comment:
Are we 100% sure this won't break any JS?

Reply:
We're never 100% sure of anything, but it shouldn't! 😅

Reply:
I'll give it more time to test.

Reply:
Okay let me know when you've tried it out and then I'll merge it. Thanks!

Reply:
Seems to be okay.

Reply:
Hey! Sorry, totally forgot to comment back. I also tested and can't find anything wrong with this change.